src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/Tranche.java

```diff
@@ -27,14 +27,16 @@ public class Tranche {
     final double minVQSLod;  //minimum value of VQSLOD in this tranche
     final double knownTiTv;  //titv value of known sites in this tranche
     final double novelTiTv;  //titv value of novel sites in this tranche
-    final int numKnown;      //number of known sites in this tranche
-    final int numNovel;      //number of novel sites in this tranche
+    final long numKnown;      //number of known sites in this tranche
+    final long numNovel;      //number of novel sites in this tranche
     final VariantRecalibratorArgumentCollection.Mode model;
     final String name;       //Name of the tranche
 
-    public Tranche(final String name, final double knownTiTv, final int numNovel, final double minVQSLod, final VariantRecalibratorArgumentCollection.Mode model, final double novelTiTv, final int accessibleTruthSites, final int numKnown, final int callsAtTruthSites) {
+    public Tranche(final String name, final double knownTiTv, final long numNovel, final double minVQSLod,
+                   final VariantRecalibratorArgumentCollection.Mode model, final double novelTiTv,
+                   final int accessibleTruthSites, final long numKnown, final int callsAtTruthSites) {
```

Comment:
Did you check the upstream call sites for uses of `int` to store the number of known/novel sites? If there's an `int` anywhere in the call chain the overflow issues will likely persist...

Reply:
I found one in the input parsing and fixed it.  (The number of truth sites values are also ints, but with the resources we use today they're way, way below Integer.MAX_VALUE and I don't expect that to ever change based on the size of the human genome.)